loops.js

```diff
@@ -30,6 +38,15 @@
 // .... etc.
 // 15 is oneven
 
+for (let i = 0; i < 16; i++) {
+    if (i % 2 === 0) {
+        console.log(`${i} is even`)
```

Comment:
Voor mij is ${i} een nieuw element, en ook nog niet gebruikt. Maar het lijkt me bij langere zinnen makkelijk te schrijven, en een beter overzicht te geven.